content/en/docs/tasks/tools/install-kubeadm.md

```diff
@@ -234,23 +234,18 @@ kubeadm to tell it what to do.
 
 ## Configure cgroup driver used by kubelet on Master Node
 
-Make sure that the cgroup driver used by kubelet is the same as the one used by Docker. Verify that your Docker cgroup driver matches the kubelet config:
+When using Docker, kubeadm will detect the cgroup driver for the kubelet
+automatically and set in the file `/var/lib/kubelet/kubeadm-flags.env`
+on runtime.
 
-```bash
-docker info | grep -i cgroup
-cat /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
-```
-
-If the Docker cgroup driver and the kubelet config don't match, change the kubelet config to match the Docker cgroup driver. The
-flag you need to change is `--cgroup-driver`. If it's already set, you can update like so:
-
-```bash
-sed -i "s/cgroup-driver=systemd/cgroup-driver=cgroupfs/g" /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
-```
+If you are using a different CRI, you have to modify the the drop-in file
+`/etc/systemd/system/kubelet.service.d/10-kubeadm.conf`, by adding
+`--cgroup-driver=<driver-name>`.
 
-Otherwise, you will need to open the systemd file and add the flag to an existing environment line.
+Please mind, that you **only** have to do that if the cgroup driver of your CRI
```

Comment:
This sentence is odd. 

I'm going to defer to @kad  to lgtm 

/approve 
/cc @kad

Reply:
for reference:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/kubeletconfig/v1beta1/defaults.go#L125-L127

```
	if obj.CgroupDriver == "" {
		obj.CgroupDriver = "cgroupfs"
	}
```